examples/db-templates/mysql-template.json

```diff
@@ -40,6 +40,23 @@
       }
     },
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1beta3",
+      "metadata": {
+        "name": "${PVC_NAME}"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteMany"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "1Gi"
```

Comment:
You beat me to it :) Question: Do we have some validation mechanisms for parameters? E.g. enforcing the correct format for storage capacity specification.